src/code-system/cts-metadata.json

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
The files in this directory are just used in the script, right? If so, we should make sure these JSON files don't get included in the `build` directory when doing `npm run build` which will increase the size of the fqm-execution package despite not being relevant for the source code